### PR TITLE
Replace deprecated `validator` usage

### PR DESCRIPTION
**Pull request by Viicos**
## Change Summary

There might be other places where this needs to be replaced, but I think I've covered all the ones. Let me know if this requires a new file in `changes`. 

## Checklist

* [ ] Unit tests for the changes exist
* [ ] Tests pass on CI and coverage remains at 100%
* [ ] Documentation reflects the changes where applicable
* [ ] `changes/<pull request or issue id>-<github username>.md` file added describing change
  (see [changes/README.md](https://github.com/samuelcolvin/pydantic/blob/master/changes/README.md) for details)
* [x] My PR is ready to review, **please add a comment including the phrase "please review" to assign reviewers**

Selected Reviewer: @hramezani

### CODE REVIEW

**Comment by Viicos** [comment]
Please review

File: docs/usage/validators.md [hunk 1]
```diff
@@ -204,13 +204,13 @@ to set a dynamic default value.
...
 
-from pydantic import BaseModel, validator
+from pydantic import BaseModel, field_validator
 
 
 class DemoModel(BaseModel):
     ts: datetime = None
 
-    @validator('ts', pre=True, always=True)
+    @field_validator('ts', pre=True, always=True)
```
Comment (hramezani) [inline docs/usage/validators.md:213]: `pre` and `always` are not valid in `field_validator`

**Comment by hramezani** [comment]
Thanks @Viicos for this.

please update

File: docs/usage/validators.md (hunk 1 above)
Comment (Viicos) [reply docs/usage/validators.md]: Reverted: https://github.com/pydantic/pydantic/pull/5712/commits/0d7cfa6543a9fe3a50a91cfe6f5acca1c95a8176

**Comment by Viicos** [comment]
Please review

**Pull request merged**